app/src/main/admin/components/Touchstones/List/TouchstoneList.tsx

```diff
@@ -16,15 +16,15 @@ export const TouchstoneListComponent: React.FunctionComponent<TouchstoneListProp
         <div className="mb-5 row">
             <div className="col-12">
                 <h4>Active touchstones</h4>
-                <div className="mb-2">The latest version of these touchstones is 'open' or 'in-preparation'.</div>
-                <TouchstoneTable touchstones={props.active}/>
+                <div className="mb-2">These touchstones have at least one version that is 'open' or 'in-preparation'.</div>
```

Comment:
ok, so in the case of touchstones which have a previous version unfinished, it still appears in the active list, but linking to the latest unfinished version?